sodapy/__init__.py

```diff
@@ -94,7 +94,7 @@ def __exit__(self, exc_type=None, exc_value=None, traceback=None):
         '''
         self.close()
 
-    def datasets(self, limit=0, offset=0):
+    def datasets(self, limit=0, offset=0, order=None, **kwargs):
```

Comment:
Why not simply eliminate limit and offset from the method signature and simply pass kwargs directly to params? While the field names are good to know, maybe we could document them as possible options for kwargs in the method comment.

Reply:
`offset` needs special treatment because we increase it to get the following pages.

`limit` could be put in `params` line 123 with the rest of those arguments I suppose, but that would break compatibility with people using positional arguments, e.g. `socrata.datasets(50)` will no longer work.